### ISSUE TITLE
[post list] Crash tapping the top right icon

### ISSUE THREAD

**Issue by NicKoehler** [diegoberaldin/RaccoonForLemmy#933]
**Describe the bug**
Tapping the fire icon in the top right crash the app

**To Reproduce**
1. Open the app
2. Tap the fire icon

**Screenshots**
![Screenshot_20240605-151747~2.png](https://github.com/diegoberaldin/RaccoonForLemmy/assets/53040044/050d6a19-f099-4731-8383-66f98cf4aa75)

**Smartphone (please complete the following information):**
 - Device: Pixel 7 Pro
 - OS: Android 14
 - Version 1.11.0

**Additional context**
```
type: crash
osVersion: google/cheetah/cheetah:14/AP1A.240505.005/2024053100:user/release-keys
package: com.github.diegoberaldin.raccoonforlemmy.android:103
process: com.github.diegoberaldin.raccoonforlemmy.android
processUptime: 1389 + 240 ms
installer: dev.imranr.obtainium

java.lang.IllegalArgumentException: Key d4.w:currentScreen was used multiple times 
	at D.m.a(Unknown Source:1293)
	at O.z.a(Unknown Source:4)
	at B2.b.q(Unknown Source:130)
	at O.t.g(Unknown Source:61)
	at O.t.f(Unknown Source:5)
	at O.o0.a(Unknown Source:203)
	at v0.T.doFrame(Unknown Source:6)
	at v0.Q.doFrame(Unknown Source:47)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1397)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1408)
	at android.view.Choreographer.doCallbacks(Choreographer.java:1008)
	at android.view.Choreographer.doFrame(Choreographer.java:934)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1382)
	at android.os.Handler.handleCallback(Handler.java:959)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loopOnce(Looper.java:232)
	at android.os.Looper.loop(Looper.java:317)
	at android.app.ActivityThread.main(ActivityThread.java:8532)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
	at com.android.internal.os.ExecInit.main(ExecInit.java:50)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:359)
	Suppressed: t9.g: [O.c0@1566d17, v0.r0@dd3a504, x0{Cancelling}@47caaed, S@5772d22]
```

**Comment by diegoberaldin** [comment]
This issue technically has been solved in version 1.11.1 released today, could you please confirm whether it happens again after the update?

**Comment by NicKoehler** [comment]
Yes, now is working
